kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -13,12 +13,12 @@ import kyo.internal.OsSignal
 abstract class KyoApp extends KyoAppPlatformSpecific:
     private val awaitInterrupt =
         given AllowUnsafe = AllowUnsafe.embrace.danger
-        val promise       = Promise.Unsafe.init[Nothing, Nothing]()
+        val promise       = Promise.Unsafe.init[Nothing, Any]()
 
         val interrupt = (signal: String) =>
             () =>
                 promise
-                    .completeDiscard(Result.panic(Interrupted(Frame.internal, s"Interrupt Signal: $signal")))
+                    .completeDiscard(Result.panic(Interrupted(Frame.internal, s"Interrupt Signal Reached: $signal")))
```

Comment:
The fiber message now says 'Fiber interrupted by $xyz' when a message is present, so this currently says: FIber Interrupted by Interrupt Signal INT - or whatever.
```suggestion
                    .completeDiscard(Result.panic(Interrupted(Frame.internal, s"Interrupt Signal: $signal")))
```

I think this is just a git rebase issue.